spec/data-model/README.md

```diff
@@ -214,11 +220,43 @@ that the implementation attaches to that _annotation_.
 ```ts
 interface UnsupportedAnnotation {
   type: "unsupported-annotation";
-  sigil: "!" | "@" | "#" | "%" | "^" | "&" | "*" | "<" | ">" | "/" | "?" | "~";
+  sigil: "!" | "@" | "%" | "^" | "&" | "*" | "+" | "<" | ">" | "?" | "~";
```

Comment:
what order is this list in? Probably we should sort the reserved sigils into code point order?

Reply:
I think it's currently something close to the layout order on a US keyboard? I've no opinion on the order here.